.bazelrc

```diff
@@ -1,3 +1,4 @@
+build --define=GO_VERSION=go1.23
```

Comment:
why do we need this? It's not good practice to use `--define` because it's hard to trace what it's actually doing. Better to use starlark flags such as `build --@aws_in_a_box//flags:go_version` or something, so you can follow them to the BUILD file.

but also if we just want to control the version of go, that should be done in MODULE.bazel